packages/pluginsdk-peerdeps/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@spinnaker/pluginsdk-peerdeps",
+  "description": "Provides package dependencies to plugin developers",
+  "version": "0.0.1",
+  "license": "Apache-2.0",
+  "scripts": {
+    "usage": "cat USAGE.txt",
+    "stage": "npm run clean && ./sync-stage.sh",
+    "sync": "./sync-save.js",
+    "clean": "./clean.sh"
+  },
```

Comment:
These package scripts are written to help maintain this package.  
They make it easier to update the `peerDependencies`